test/jdk/javax/swing/text/html/HTMLDocument/bug4209280.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 1999, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javax.swing.JLabel;
+
+/*
+ * @test
+ * @bug 4209280
+ * @summary Tests that no exception is thrown on unknown HTML tag
+ */
+
+public class bug4209280 {
+
+    public static void main(String[] args) throws Exception {
+        String html = "<html><bold>Foo</bold></html>";
+        new JLabel(html);
```

Comment:
```suggestion
        // The following line should throw no exceptions
        new JLabel(html);
```
This may clarify the intention of the reader skipped the summary.

Reply:
Usually full stops aren't used in comments, unless it consists of several sentences. I'm nitpicking.